packages/terra-avatar/src/terra-dev-site/doc/avatar/UpgradeGuide.5.doc.mdx

```diff
@@ -0,0 +1,90 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+<Badge />
+
+# terra-avatar Upgrade Guide
```

Comment:
It looks like we copied over the content from docs/upgrade-guide.md/ Do we need to delete `'../../../../docs/upgrade-guide.md';`?

Reply:
updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6